app/views/checkout/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Welcome to checkout</h2>
+<% @cart.shopping_cart_items.each do |cart| %>
+  <div class="images">
```

Comment:
If the reason for that is to reuse the step definition that is an `id` you need, not `class`